### PR TITLE
Alignment with other CCAMP YANG models

### PR DESCRIPTION
**Pull request by italobusi**
Imported ietf-te and ietf-te-types by revision: see [otn-tunnel PR#71](https://github.com/haomianzheng/IETF-ACTN-YANG-Model/pull/71)

Imported path-computation by revision: aligned with ietf-te and ietf-te-types revisions

Imported l0-types-ext by revision, pending merging of [l0-types-ext PR#17](https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/pull/17)

Removed src-client-signal and dst-client-signal: see [otn-tunnel PR#77](https://github.com/haomianzheng/IETF-ACTN-YANG-Model/pull/77)

Removed TE bandwidth augmentations: see [l0-types Issue#8](https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-layer0-types/issues/8)

Aligned TE label augmentations with the template used in the latest version of the OTN tunnel: see [otn-tunnel PR#71](https://github.com/haomianzheng/IETF-ACTN-YANG-Model/pull/71)

Editorial clean-up

### CODE REVIEW

**Comment by italobusi** [comment]
@danielkinguk : I cannot ask Daniel Perdices to review this pull request in GH (we need to invite him to join as contributor to the GH repository)

File: ietf-layer0-types-ext@2021-02-04.yang [hunk 1]
```diff
@@ -0,0 +1,813 @@
...
+    leaf rx-total-power {
+        type dbm-t;
+        config false;
+        description "Current total received power";
+    }
+
+  } // grouping for configured attributes out of mode
+
+
+} // end of module
```
Comment (agva123) [inline ietf-layer0-types-ext@2021-02-04.yang:813]: This comment needs to be removed or yanglint will throw error:
err : Trailing garbage "// end of modul..." after module, expected end-of-input. (Line number 813.)
err : Loading "ietf-layer0-types-ext" module failed.

**Review by agva123 — APPROVED** [review]
Model alignment changes of flex-grid-yang with ietf-te looks good with me. Compilation with pyang and yanglint passed with no warnings.

**Comment by dperdices** [comment]
I think this can be merged. I couldn't validate the media channel yang on this [page](https://yangcatalog.org/yangvalidator/validator) but it seems that others have already done so. 

There are still some minor things about the format that can be addressed later (descriptions, the prefix of the document, etc.) but, as an update, it is ready in my opinion. Thanks, Italo!

**Pull request merged**